src/seedu/addressbook/parser/Parser.java

```diff
@@ -217,7 +221,7 @@ private Command prepareViewAll(String args) {
      *
      * @param args arguments string to parse as index number
      * @return the parsed index number
-     * @throws ParseException if no region of the args string could be found for the index
+     * @throws ParseException        if no region of the args string could be found for the index
```

Comment:
Extra whitespace added by accident?

Reply:
It is the default auto reformat option that intellij does. It is slightly different from the default address book indentation style, I will take note of that!